integreat_cms/cms/migrations/0016_rename_location_not_on_map_poi_location_on_map.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.12 on 2022-04-03 14:22
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0015_user_unique_email_field'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='poi',
+            old_name='location_not_on_map',
+            new_name='location_on_map',
+        ),
+    ]
```

Comment:
```suggestion
        migrations.AlterField(
            model_name='poi',
            name='location_on_map',
            field=models.BooleanField(default=True, help_text='Tick if you want to show this location on map', verbose_name='Show this location on map'),
        ),
    ]
```

> Actually I think I messed around a little too much with the migration files. I wasn't sure what they were doing at first, so I did a bit of unnecessary back and forth editing. I tried to get things as clean as possible, but I always ended up creating at least 2 migration files. Is that right?


You can combine the two migration files by moving the operation of the second file into the first one and then deleting the second file.